README.md

```diff
@@ -131,7 +143,30 @@ entities (no need to draw an ERD):
 
 Your answer:
 ```
-Replace this with your answer
+Genie
+- Name
+- has_one :Lamp
+- wishes: wishes_granted
+
+Lamp
+- Genie ID
+- made_of :metal
+- contains: Genie
+- belongs_to :Genie
+
+Person
+-Name
+-Age
+- owns_one :pet
+
+Pet
+-Person ID
+-Name
+-Breed
+-belongs_to :Person
```

Comment:
You have two entire that indicate a Per belongs_to a Person: foreign_key ("Person ID") and a Rails helper that creates that foreign key (belongs_to :person).